### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by onekatytocode**
finished bits

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -4,36 +4,21 @@
...
 	background-color: #986960;
+	padding: 10px;
 	font-family: 'Raleway', sans-serif;
 	font-family: Raleway;
 	src: url("raleway");
 	background: linear-gradient(to bottom,
 		#986960 0%,
-		#986960 80%,
-		#ffffff 80%,
+		#986960 70%,
```
Comment (PaulREnglish) [inline style.css:12]: If you have this, I don't think background-color is needed.  Also, there doesn't semm to be any gradients appearing on your webpage.

File: index.html [hunk 2]
```diff
@@ -1,75 +1,73 @@
...
-	<link rel="stylesheet" href="normalze.css" type="text/css" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="preconnect" href="https://fonts.gstatic.com">
-	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
-	<script src="https://kit.fontawesome.com/a5bc471788.js" crossorigin="anonymous"></script>
-	
+<title>Home page</title>
+<link rel="stylesheet" href="style.css" type="text/css" />
+<link rel="stylesheet" href="navbar.css" type="text/css" />
+<link rel="stylesheet" href="normalize.css" type="text/css" />
```
Comment (PaulREnglish) [inline index.html]: This stylesheet needs to be loaded first to prevent it from overriding your other stylesheets.

File: navbar.css [hunk 3]
```diff
@@ -2,39 +2,22 @@ nav {
...
-	color: #ffffff;
-	
+	color: #ffe4e1;
 }
 
+li a {
+	color: #ffe4e1;
+}
 a:visited {
 	color: #ffe4e1;
```
Comment (PaulREnglish) [inline navbar.css]: I don't think you need this when you've already set the links to be in this color within the li a selector

File: navbar.css [hunk 4]
```diff
@@ -2,39 +2,22 @@ nav {
 	display: flex;
 	flex-direction: row-reverse;
 	justify-content: space-between;
-}
-
-ul {
-	
-	color: black;
+	color: #ffe4e1;
```
Comment (PaulREnglish) [inline navbar.css:5]: Does nothing.

File: navbar.css [hunk 5]
```diff
@@ -2,39 +2,22 @@ nav {
...
 }
 
 li {
 	padding: 0.5em;
 	display: inline;
-	color: #ffffff;
-	
+	color: #ffe4e1;
```
Comment (PaulREnglish) [inline navbar.css:11]: Does nothing

File: style.css [hunk 6]
```diff
@@ -71,27 +58,35 @@ h2 {
 	text-align: center;
 	font-weight: bold;
 }
-/* .item {
-	flex-grow: 3; /* might need to set a width for child elements */
+
+h1 {
+	font-size: 3.1rem;
+	color: #ffe4e1;
```
Comment (PaulREnglish) [inline style.css]: I think you've set the sane color in the nav selector.  So either keep this or keep the color in the nav selector but not both.

File: navbar.css [hunk 7]
```diff
@@ -2,39 +2,22 @@ nav {
 	display: flex;
 	flex-direction: row-reverse;
```
Comment (PaulREnglish) [inline navbar.css:3]: I'd take this out and then change the order of the elements within the html document.

File: normalize.css [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
+
+/* Embedded content
+   ========================================================================== */
+
+/**
+ * Remove the border on images inside links in IE 10.
+ */
+
+img {
+  border-style: none;
```
Comment (PaulREnglish) [inline normalize.css:149]: Does nothing.

File: navbar.css (hunk 7 above)
Comment (onekatytocode) [reply navbar.css:3]: This is the only one I have issues with, when I change it it messes up, I have tried though!

**Pull request merged**